faction_reputation/federation.json

```diff
@@ -0,0 +1,177 @@
+[
+  {
+    "dailies_barrier": false,
+    "locked": false,
+    "points_min": -2000000,
+    "reputation_id": 1,
+    "reputation_name": ""
```

Comment:
There is a field empty here, same for rom and klings

Reply:
There is no name in the game for that level. the old bot called it `max negative` but that felt kinda strange so I just omitted the name for that level. If you have an idea for a good name I can add that instead.

Reply:
I've created an issue for it, lets solve this later.